docs/contributing.md

```diff
@@ -26,12 +26,31 @@ When developing changes to packages within this repository that a different proj
 If you're a MetaMask contributor, you can create these preview versions via draft pull requests:
 
 1. Navigate to your settings within GitHub and [create a classic access token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token#creating-a-personal-access-token-classic). Make sure to give this token the `packages:read` scope.
-2. Switch to your project locally and add a `.npmrc` file with the following content, filling in the appropriate areas:
-   ```
-   @metamask:registry=https://npm.pkg.github.com
-   //npm.pkg.github.com/:_authToken=<your personal access token>
-   ```
+2. Switch to your project locally and add/edit the appropriate file with the following content, filling in the appropriate areas:
+
+   - **Yarn 1 (classic) or NPM**
+
+     Add the following in `.npmrc`
+
+     ```
+     @metamask:registry=https://npm.pkg.github.com
+     //npm.pkg.github.com/:_authToken=<your personal access token>
+     ```
+
+   - **Yarn >= 2 (berry):**
+
+     Add the following in `.yarnrc.yml`
+
+     ```
+     npmScopes:
+        metamask:
+           npmAlwaysAuth: true
+           npmAuthToken: <your personal access token>
+           npmRegistryServer: 'https://npm.pkg.github.com'
+     ```
+
    Make sure not to commit this file.
```

Comment:
Since `.yarnrc.yml` is in the repo for Yarn 2+ projects, maybe this should now say:

``` suggestion
   Make sure not to commit these changes.
```